pkg/models/plan_data_exchange_approach.go

```diff
@@ -0,0 +1,128 @@
+package models
+
+import (
+	"github.com/lib/pq"
+)
+
+// PlanDataExchangeApproach represents the data exchange approach of a model plan
+type PlanDataExchangeApproach struct {
+	coreTaskListSection
+	markedCompleteByRelation
+
+	// Page 2
+	DataToCollectFromParticipants               pq.StringArray `json:"dataToCollectFromParticipants" db:"data_to_collect_from_participants" statusWeight:"1"`
+	DataToCollectFromParticipantsReportsDetails *string        `json:"dataToCollectFromParticipantsReportsDetails" db:"data_to_collect_from_participants_reports_details"`
+	DataToCollectFromParticipantsOther          *string        `json:"dataToCollectFromParticipantsOther" db:"data_to_collect_from_participants_other"`
+	DataWillNotBeCollectedFromParticipants      bool           `json:"dataWillNotBeCollectedFromParticipants" db:"data_will_not_be_collected_from_participants" statusWeight:"1"`
```

Comment:
I think this was only updated on the front end , but this bool should also be a pointer ( and can then be removed from the instantiation)